### PR TITLE
feature: UMD bundle example

### PR DESCRIPTION
**Pull request by phamhieu**
## What kind of change does this PR introduce?

update docs with UMD bundle example

## What is the new behavior?

Screenshot 2020-08-28 at 5 08 26 PM

## Additional context

#155

### CODE REVIEW

File: web/src/pages/index.js [hunk 1]
```diff
@@ -107,6 +107,29 @@ export default async (req: NextApiRequest, res: NextApiResponse) => {
...
+  // Get public rooms and their messages
+  supabase
+    .from('rooms')
+    .select(\`
+      name,
+      messages ( text )
+    \`)
+    .eq('public', true)
+    .then(response => {
+      // do s.th with the response
```
Comment (kiwicopple) [inline web/src/pages/index.js:128]: Can we make this one "Do something with the response"?

File: web/src/pages/index.js [hunk 2]
```diff
@@ -371,6 +394,14 @@ function Home() {
                     >
                       Node.js & TypeScript support
                     </button>
+                    <button
+                      className={`button button--${
+                        visibleCodeExample === 'UMD' ? 'info is-active' : 'info '
+                      }`}
+                      onClick={() => showCodeExample('UMD')}
+                    >
+                      UMD bundle support
```
Comment (kiwicopple) [inline web/src/pages/index.js:403]: Can we make this one "Using on a website" or "With HTML"

I think there is probably a better term for this, but I can't think of it right now. I'm not sure how many "html devs" will actually know what UMD is though, so we should definitely try to use a phrase which is more approachable

**Review by kiwicopple — APPROVED** [review]

File: web/src/pages/index.js (hunk 2 above)
Comment (awalias) [reply web/src/pages/index.js:403]: "HTML snippet example" or "CDN example"?

Comment (phamhieu) [reply web/src/pages/index.js:403]: I change it to "Install from CDN". How's it?

Comment (kiwicopple) [reply web/src/pages/index.js]: I think it's still not entirely accurate in describing what it's doing. Technically installing from `npm` is installing from a CDN 😂 

But it's fine for now. Ship first and if we think of a more accurate term we can reword it

**Pull request merged**